src/session/sessionCookieStore.ts

```diff
@@ -1,39 +1,30 @@
-import { simpleSessionCookieStore } from './app-session-cookie-store'
-import { shouldRefresh } from './shouldRefresh/shouldRefresh'
-import { refreshAppSession } from './refreshAppSession/refreshAppSession'
-import { refreshToken } from '../storyblok-auth-api/refreshToken'
-import { AppSessionCookieStoreFactory } from './types'
-import { AppSessionStore } from './types'
+import { AppSessionCookieStoreFactory, AppSessionStore } from './types'
+import { getAllSessions, getSession, putSession, removeSession } from './crud'
+import { refreshStoredAppSession } from './refreshStoredAppSession'
+import {
+  GetCookie,
+  getCookie as getNodeCookie,
+  SetCookie,
+  setCookie as setNodeCookie,
+} from '../utils'
 
 export const sessionCookieStore: AppSessionCookieStoreFactory =
   (params) =>
   (requestParams): AppSessionStore => {
-    const store = simpleSessionCookieStore(params)(requestParams)
+    const getCookie: GetCookie = (name) =>
+      getNodeCookie(requestParams.req, name)
+    const setCookie: SetCookie = (name, value) =>
+      setNodeCookie(requestParams.res, name, value)
```

Comment:
So far, from my understanding this `sessionCookieStore` is still relying on the Node.js request and response object because `getNodeCookie` and `setNodeCookie` expects them. And your plan to going to make this replaceable? Is it the changes you told me you're thinking of?

Reply:
Yes exactly. 

It also makes the other functions easier to test, because we don't need to mock request and response objects; just the getter and setter functions.